src/main/java/racingcar/common/UserInput.java

```diff
@@ -0,0 +1,93 @@
+package racingcar.common;
+
+import racingcar.common.exception.InputValidationException;
+
+import java.io.*;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static racingcar.common.SystemMessage.*;
+
+public class UserInput {
+
+    private static final String DELIMITER = ",";
+    private static final int LENGTH_LIMIT_MAX = 5;
+    private static final int LENGTH_LIMIT_MIN = 1;
+
+    private List<String> splitUserInput;
+    private String carNameInput;
+    private int racingTime;
+
+    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+    BufferedWriter log = new BufferedWriter(new OutputStreamWriter(System.out));
+
+    private UserInput() {
+        while (parsingCarName())
+            ;
+        while (parsingRacingTime())
+            ;
+    }
+
+    public static UserInput of() {
```

Comment:
파라미터가 없을 땐 `of` 보단 `newInstence` or `instence`가 좀 더 나은 표현이라 생각합니다.
```suggestion
    public static UserInput newInstence() {
```

Reply:
정적 팩토리 메서드 네이밍을 변경하는 과정에서 누락되었네요. 감사합니다!!

다른 정적 팩토리 메서드와 일관성을 위해 `instance`로 변경하겠습니다!